docs/src/content/docs/development/testing.md

```
@@ -0,0 +1,113 @@
+# Run specific test file
+flutter test test/battery_test.dart
+
+# Run with coverage
+flutter test --coverage
+```
+
+## Test Structure
+
+Tests are located in the `test/` directory mirroring the lib structure:
+
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)